packages/domain/lib/use_case/login_validator_use_case.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:domain/model/firebase_user_email.dart';
+import 'package:domain/model/validate_model.dart';
+import 'package:domain/use_case/sample_use_case/use_case_in_out.dart';
+
+const _patternRegexLogin = '^.{8,}\$';
+const _patternRegexPassword = '^[^A-Z][^a-z]{6,}\$';
+
+class ValidatorUseCase extends UseCaseInOut<UserEmailPass, ValidateModel> {
+  @override
+  ValidateModel call(UserEmailPass params) {
+    return ValidateModel(
+      validIsEmptyLogin: validIsEmptyLogin(params.login),
+      validRegexLogin: validRegexLogin(params.login),
+      validIsEmptyPassword: validIsEmptyPassword(params.password),
+      validRegexPassword: validRegexPassword(params.password),
```

Comment:
1) rename the ValidateModel into LoginValidationModel
2) no need to return the result of all checks back to bloc. it shouldn't know about it. return just the failed validators for the each field